azure-pipelines.yml

```diff
@@ -42,280 +33,236 @@ resources:
-            XUnitRunnerVersion: 2.4.2-pre.9
-            XUnitPublishTargetFramework: net8.0
-            IncludeDotNetCli: true
-            DotNetCliPackageType: sdk
-            DotNetCliVersion: 8.0.100
-            EnableXUnitReporter: true
-            WaitForWorkItemCompletion: true
-            ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-              HelixTargetQueues: Windows.10.Amd64.Open;(Debian.10.Amd64.Open)Ubuntu.2004.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-10-helix-amd64
-              HelixSource: pr/dotnet/arcade-validation/$(Build.SourceBranch)
-              IsExternal: true
-              Creator: arcade-validation
-            ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-              HelixTargetQueues: Windows.10.Amd64;(Debian.10.Amd64)Ubuntu.2004.Amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-10-helix-amd64
-              HelixSource: official/dotnet/arcade-validation/$(Build.SourceBranch)
-              HelixAccessToken: $(HelixApiAccessToken)
-        displayName: Validate Helix
+      - template: /eng/common/templates-official/jobs/jobs.yml@self
+        parameters:
+          enableMicrobuild: true
+          enablePublishBuildArtifacts: true
+          enablePublishBuildAssets: true
+          enablePublishUsingPipelines: ${{ variables._PublishUsingPipelines }}
+          enableTelemetry: true
+          helixRepo: dotnet/arcade-validation
+          jobs:
+          - job: Windows_NT
+            variables:
+            - _BuildConfig: Release
+            - _SignType: real
+            - _InternalBuildArgs: /p:DotNetSignType=$(_SignType)
+                /p:TeamName=$(_TeamName)
+                /p:DotNetPublishUsingPipelines=$(_PublishUsingPipelines)
+                /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
 
-  # Jobs that should only run as part of internal builds.
-      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-        - job: Validate_Signing
-          pool:
-            name: $(PoolProvider)  # This is a queue-time parameter; Public default is NetCore-Svc-Public; Internal default is NetCore1ESPool-Svc-Internal
-            demands: ImageOverride -equals windows.vs2019.amd64
-          strategy:
-            matrix:
-              Test_Signing:
-                _BuildConfig: Debug
-                _SignType: test
-              Real_Signing:
-                _BuildConfig: Release
-                _SignType: real
-          steps:
+            steps:
             - checkout: self
               clean: true
-            - task: CopyFiles@2
-              displayName: Copy test packages to artifacts directory
+            - task: PowerShell@2
+              displayName: Setup Private Feeds Credentials
+              condition: eq(variables['Agent.OS'], 'Windows_NT')
```

Comment:
nit: this `condition` is unnecessary. same for a similar Linux condition not too much further down. they're in os-specific jobs.

Reply:
fixed :)